src/amo/components/InstallButtonWrapper/styles.scss

```diff
@@ -15,6 +15,6 @@
   a:link {
     color: $sub-text-color;
     font-size: $font-size-s;
-    font-weight: lighter;
+    font-weight: 300;
```

Comment:
just pointing out we're changing a relative `font-weight` to an absolute here.  (And that 300 is the correct weight from whatever style guide we're adhering to)

Reply:
I don't see any specific font-weights in the design guide that would apply to this. I just chose one that is light, but not so light that it becomes impossible to see the link.